data/json/itemgroups/vending_machines.json

```
@@ -130,5 +130,303 @@
     "subtype": "collection",
     "id": "vending_medicine",
     "entries": [ { "group": "drugs_vending", "count-min": 10, "count-max": 20 } ]
+  },
+  {
+    "type": "item_group",
+    "id": "vending_items_adult",
+    "items": [
+      { "item": "condom", "prob": 30, "count": [ 1, 5 ] },
+      { "group": "ss_merch_1", "prob": 20 },
+      { "group": "ss_merch_2", "prob": 20 },
+      { "group": "ss_merch_3", "prob": 20 },
+      { "item": "mag_porn", "prob": 20, "count": [ 1, 3 ] }
+    ]
+  },
+  {
+    "type": "item_group",
+    "id": "vending_items_useful",
+    "items": [
+      { "item": "umbrella", "prob": 10 },
+      { "item": "teleumbrella", "prob": 10 },
+      { "group": "home_hw", "prob": 50 },
+      { "group": "tools_common", "prob": 50 },
+      { "group": "cleaning", "prob": 30 }
```

`home_hw` again contains lots of bulky items, notably entire lawnmowers and empty trash cans. General `cleaning` itemgroup meanwhile contains a wide variety of stuff, including clothing, buckets, mops, etc.
```suggestion
      { "item": "pen", "count": [ 1, 5 ], "prob": 50 },
      { "item": "usb_drive", "prob": 30 },
      { "group": "tools_drawer", "prob": 50 },
      { "group": "tools_cleaning_small", "prob": 30 }
```